xml_handlers/testing_handler.php

```diff
@@ -132,7 +132,7 @@ public function startElement($parser, $name, $attributes)
             $this->Label = new Label();
         } elseif ($name == 'TESTLIST' && $parent == 'TESTING') {
             $start_time = gmdate(FMT_DATETIME, $this->StartTimeStamp);
-            $end_time = gmdate(FMT_DATETIME, $this->EndTimeStamp); // The EndTimeStamp
+            // $end_time = gmdate(FMT_DATETIME, $this->EndTimeStamp); // The EndTimeStamp
```

Comment:
You would think so, but people seem to have been relying on this in tests. Any good reason for it?